README.md

```diff
@@ -94,19 +96,20 @@ ex.
 
 ##### Build the client
 
-As an alternative to downloading the client build the client from source.
-
+In the root of the project:
 
 ```
 git submodule update --recursive --init
 sbt dist
 ```
 
-in the root of the project.
-
 This updates all submodules and creates a distribution zip in `~/target/universal/`.
 
-#### Nix
+Note: building in _dev_ mode allows faster and incremental compilation, for this:
+   - set environment variable `MANTIS_DEV` to `true`, or
+   - use the compiler option `-DmantisDev=true`
```

Comment:
Nitpick: this is not a compiler option but a "system property value."

Reply:
You're right 😿